### PR TITLE
Update meeting minutes

### PR DESCRIPTION
**Pull request by kalbermattenm**
Just some small updates.

@rbovard, ok for you ?

### CODE REVIEW

File: content/pages/meetings/2017-11-06_fr.md [hunk 1]
```diff
@@ -20,9 +20,13 @@ Status: hidden
...
 * ...
 
+### Les Iframe pour remplacer l'API (Peter Gabathuler)
+
+* ...
+
 ### Divers
 
-* Charte et site web (Luis Teba et Michael Kalbermatten)
+* Site web (Rémi Bovard et Michael Kalbermatten)
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: Me?

File: content/pages/meetings/2017-11-06_fr.md [hunk 2]
```diff
@@ -20,9 +20,13 @@ Status: hidden
 
 * ...
 
+### Les Iframe pour remplacer l'API (Peter Gabathuler)
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: Iframe"**s**" or remove it in german title?

**Review by rbovard — COMMENTED** [review]
Thanks, just 2 remarks.

**Comment by kalbermattenm** [comment]
@rbovard, could just have a look at the minutes in French ? I feel that I forgot something, but I cannot get what...

(I will add the German ones this afternoon)

Thanks ;-)

File: content/pages/meetings/2017-11-06_fr.md [hunk 3]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
 
-* ...
+* Pour la partie ne concernant pas l'interface d'administration, 80 % des développements ont été effectuées
+* Pour la partie interface d'administration, un gros de travail de fond a été effectué. Cela va induire un léger retard de livraison (mi-février)
 
 ### Impact de la suppression de Google Closure Compiler dans OpenLayers
 
-* ...
+Camptocamp se propose de prendre à sa charge 30% des jours nécessaires à ce travail. Pour le reste une offre concrète va être faite de la part de Camptocamp au groupe GMF.
+Suivant les conditions exposées, le groupe est d'accord d'effecturer les modifications nécessaires à la suppression de Google Closure Compiler.
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: Missing `* ` (2x)

File: content/pages/meetings/2017-11-06_fr.md [hunk 4]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
 
 ### Etat de la version GMF 2.4 (PSC)
 
-* ...
+Le PSC et Camptocamp présente les développements pour lesquels une offre va être demandée. Ils feront partis de la future version 2.4.
+
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: rm empty line

File: content/pages/meetings/2017-11-06_fr.md [hunk 5]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
+La version et ses composants sont décrits sur la page de la Roadmap.
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: Add link?

File: content/pages/meetings/2017-11-06_fr.md [hunk 6]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
 Lang: fr
 Status: hidden
 
+<br><br>
+Les trois prochains points sont repris dans la présentation de Camptocamp, [gmfusrgrp_20171106.pdf]({filename}/documents/meetings/2017-11-06/gmfusrgrp_20171106.pdf) 
+
 ### Etat de la version GMF 2.3 (Camptocamp)
 
-* ...
+* Pour la partie ne concernant pas l'interface d'administration, 80 % des développements ont été effectuées
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: `80 %` > `80%`?

File: content/pages/meetings/2017-11-06_fr.md [hunk 7]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
+
+La version et ses composants sont décrits sur la page de la Roadmap.
 
 ### Les Iframe pour remplacer l'API (Peter Gabathuler)
 
-* ...
+Peter Gabathuler (BS) présente les développements qui ont été effectué par BS pour la mise en place d'Iframe, voir [20171106_PSC_UG_iFrame.pdf]({filename}/documents/meetings/2017-11-06/20171106_PSC_UG_iFrame.pdf) 
+Ces derniers peuvent remplacer l'actuelle API.
+
+Le groupe décide de rendre générique les développements de base. Une offre dans ce sens va être demandée et ce développement sera intégré à la version 2.4
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: Use a list and rm empty line

File: content/pages/meetings/2017-11-06_fr.md [hunk 8]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
-* Autre
-    * ...
+* Site web: l'ancien site web Google a disparu et il est remplacé par [http://geomapfish.org](http://geomapfish.org). Rémi Bovard a migré tous les anciens PV. Il se trouvent la page de [communauté](http://geomapfish.org/fr/community).
 
 ### Agenda
 
 * Prochaine séance
-    * Yverdon-les-Bains, ... de 13h15-16h00
+    * Yverdon-les-Bains, 12 mars 2018 de 13h15-16h00
+
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md:48]: rm empty line

File: content/pages/meetings/2017-11-06_fr.md [hunk 9]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
+Le groupe décide de rendre générique les développements de base. Une offre dans ce sens va être demandée et ce développement sera intégré à la version 2.4
 
 ### Divers
 
-* Site web (Rémi Bovard et Michael Kalbermatten)
-    * ...
-* Autre
-    * ...
+* Site web: l'ancien site web Google a disparu et il est remplacé par [http://geomapfish.org](http://geomapfish.org). Rémi Bovard a migré tous les anciens PV. Il se trouvent la page de [communauté](http://geomapfish.org/fr/community).
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: Use a sublist (2x) with
```
    *
```

File: content/pages/meetings/2017-11-06_fr.md [hunk 10]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
 Lang: fr
 Status: hidden
 
+<br><br>
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md:11]: I'm not a big fan of `<br><br>`...

File: content/pages/meetings/2017-11-06_fr.md (hunk 3 above)
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: `d'effectruer ` > `d'effectuer `

File: content/pages/meetings/2017-11-06_fr.md [hunk 11]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
 ### Impact de la suppression de Google Closure Compiler dans OpenLayers
 
-* ...
+Camptocamp se propose de prendre à sa charge 30% des jours nécessaires à ce travail. Pour le reste une offre concrète va être faite de la part de Camptocamp au groupe GMF.
+Suivant les conditions exposées, le groupe est d'accord d'effecturer les modifications nécessaires à la suppression de Google Closure Compiler.
 
 ### Etat de la version GMF 2.4 (PSC)
 
-* ...
+Le PSC et Camptocamp présente les développements pour lesquels une offre va être demandée. Ils feront partis de la future version 2.4.
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: `partis` > `partie`

File: content/pages/meetings/2017-11-06_fr.md [hunk 12]
```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
...
+
+La version et ses composants sont décrits sur la page de la Roadmap.
 
 ### Les Iframe pour remplacer l'API (Peter Gabathuler)
 
-* ...
+Peter Gabathuler (BS) présente les développements qui ont été effectué par BS pour la mise en place d'Iframe, voir [20171106_PSC_UG_iFrame.pdf]({filename}/documents/meetings/2017-11-06/20171106_PSC_UG_iFrame.pdf)
```
Comment (rbovard) [inline content/pages/meetings/2017-11-06_fr.md]: `effectué` > `effectués`

**Review by rbovard — CHANGES_REQUESTED** [review]
Thanks @kalbermattenm 

Some little style changes and typos to fix but ok for me.

According to my notes, we also talk about:
* 3D in Luxembourg, in parrallel of ngeo
* Migration to Webpack, some preliminary work done in CAS project

**Comment by kalbermattenm** [comment]
> * Migration to Webpack, some preliminary work done in CAS project

I do not get that... What is the CAS project ??

File: content/pages/meetings/2017-11-06_fr.md (hunk 10 above)
Comment (kalbermattenm) [reply content/pages/meetings/2017-11-06_fr.md:11]: I know... But otherwise the whole thing sticks to the top line (which is the language switcher)

**Comment by rbovard** [comment]
Club alpin suisse

File: content/pages/meetings/2017-11-06_fr.md (hunk 10 above)
Comment (rbovard) [reply content/pages/meetings/2017-11-06_fr.md:11]: [I know](https://github.com/geomapfish/geomapfish_website/issues/15#issuecomment-312617702)... Let's do it as it is

**Comment by kalbermattenm** [comment]
Right... I knew I was missing something... ;-)

**Comment by kalbermattenm** [comment]
@rbovard, could you tell me if OK for you ?

I updated the Roadmap as well

**Comment by rbovard** [comment]
Thanks a lot! Just update the modified date of the meeting pages and please merge :)

**Comment by kalbermattenm** [comment]
Done !

Thanks a lot for the review ! Bon week-end !!

Merging

**Pull request merged**